pyV2DL3/eventdisplay/EventDisplayDataSource.py

```diff
@@ -43,12 +44,15 @@ def __fill_response__(self, **kwargs):
                 "Parameters used to query IRFs:"
                 " az={0:.2f} deg,"
                 " ze={1:.2f} deg,"
-                " pedvar={2:.1f},"
-                " offset={3:.2f} deg"
+                " pedvar={2:.1f}"
             ).format(
-                self.__azimuth__, self.__zenith__, self.__pedvar__, self.__offset__
+                self.__azimuth__, self.__zenith__, self.__pedvar__,
```

Comment:
Help me again with why you want to remove this printout here and add it below again.

Reply:
I removed it because this printout is about "Parameters used to query IRFs", and we query and then store IRFs for all available offset (see [here](https://github.com/VERITAS-Observatory/V2DL3/blob/1181214225f03c2d701ad30b0ef54a700e360e90/pyV2DL3/eventdisplay/fillRESPONSE.py#L92)) and not just one, so I think printing it here could be misleading.

Reply:
But it is used in the call to fillResponse, or?

Reply:
we use all offset (0 to 2 deg) in fillResponse, not just one specific to given run. Also, you see that the wobble of given run was not read before anywhere.

Reply:
Please remove also self.__offset in line 18 in EventDisplayDataSource.py

If I understand it correctly, this is not needed anymore.